pkg/stringset/stringset.go

```diff
@@ -24,13 +24,31 @@ import (
 )
 
 type Interface interface {
+	// Add adds the given string to the set, if it’s not already present.
+	// The returned value is true if the string was added, false if it was already present.
 	Add(s string) bool
+
+	// AddAll adds all the given strings to the set.
```

Comment:
```suggestion
	// AddAll the given strings to the set.
```

Reply:
`godoc` comments are supposed to start with the name of the element being described, treated as a noun. See https://go.dev/blog/godoc and various examples such as https://pkg.go.dev/k8s.io/apimachinery@v0.22.2/pkg/util/cache#LRUExpireCache.Remove